File: Qubit Server/dev/src/org/labrad/qubits/mem/StartTimerCommand.java
package org.labrad.qubits.mem;

import org.labrad.qubits.FpgaModelDac;

public class StartTimerCommand implements MemoryCommand {
  private StartTimerCommand() {}

  private static final StartTimerCommand INSTANCE = new StartTimerCommand();

  public static StartTimerCommand getInstance() {
    return INSTANCE;
  }

  public long[] getBits() {
    return new long[] {0x400000};
  }
  public double getTime_us(FpgaModelDac dac) {
    return FpgaModelDac.clocksToMicroseconds(1);
  }
}

File: QubitServer/src/main/scala/org/labrad/qubits/mem/StartTimerCommand.scala
package org.labrad.qubits.mem

import org.labrad.qubits.FpgaModelDac

object StartTimerCommand extends MemoryCommand {
  def cmdBits: Array[Long] = {
    Array[Long](0x400000)
  }

  def time_us(dac: FpgaModelDac): Double = {
    FpgaModelDac.clocksToMicroseconds(1)
  }
}
